docs/src/pages/[platform]/theming/default-theme/typography/web.mdx

```diff
@@ -0,0 +1,97 @@
+import { Tabs, TabItem, View } from '@aws-amplify/ui-react';
+import { TokenList } from '@/components/Theming/TokenList';
+import { Example, ExampleCode } from '@/components/Example';
+import { FontWeightExample, FontSizesExample, LineHeightsExample } from './examples';
+
+## Font weights
+<Tabs>
+  <TabItem title="Example usage">
+    <View marginTop="var(--amplify-space-large)">
+      <Example>
+        <FontWeightExample  />
+        <ExampleCode>
+```tsx file=./examples/FontWeightExample.tsx
+
+```
+        </ExampleCode>
+      </Example>
+    </View>
+  </TabItem>
+  <TabItem title="CSS">
+    <View marginTop="var(--amplify-space-large)">
+    <Example>
+      <ExampleCode>
+```css file=./examples/FontWeightExample.css
+
+```
+      </ExampleCode>
+    </Example>
+    </View>
+  </TabItem>
+</Tabs>
+
+<TokenList namespace="fontWeights" />
+
+## Font sizes
+
+<Tabs>
+  <TabItem title="Example usage">
+    <View marginTop="var(--amplify-space-large)">
+      <Example>
+        <FontSizesExample  />
+        <ExampleCode>
+```tsx file=./examples/FontSizesExample.tsx
+
+```
+        </ExampleCode>
+      </Example>
+    </View>
+  </TabItem>
+  <TabItem title="CSS">
+    <View marginTop="var(--amplify-space-large)">
+    <Example>
+      <ExampleCode>
+```css file=./examples/FontSizesExample.css
+
+```
+      </ExampleCode>
+    </Example>
+    </View>
+  </TabItem>
+</Tabs>
+
+<TokenList namespace="fontSizes" />
+
+## Line heights
+
+<Tabs>
+  <TabItem title="Example usage">
+    <View marginTop="var(--amplify-space-large)">
+      <Example>
+        <LineHeightsExample  />
+        <ExampleCode>
+```tsx file=./examples/LineHeightsExample.tsx
+
+```
+        </ExampleCode>
+      </Example>
+    </View>
+  </TabItem>
+  <TabItem title="CSS">
+    <View marginTop="var(--amplify-space-large)">
+    <Example>
+      <ExampleCode>
+```css file=./examples/LineHeightsExample.css
+
+```
+      </ExampleCode>
+    </Example>
+    </View>
+  </TabItem>
+</Tabs>
+
+<TokenList namespace="lineHeights" />
+
+## Font families
+<TokenList namespace="fonts" />
```

Comment:
Wondering if we should include a basic explanation of the difference between static and variable fonts? Link to MDN?

Reply:
Good idea, added a note to explain what they are with a link to MDN.